packages/itwinui-variables/src/themes/dark.scss

```diff
@@ -47,23 +82,73 @@ $opacities: (
-
-  // Only HSL channels, for customizing transparency
-  --iui-color-background-1-hsl: #{get-hsl(map.get($base-colors, background-1))};
-  --iui-color-foreground-1-hsl: #{get-hsl(map.get($base-colors, foreground-1))};
-  --iui-color-foreground-primary-hsl: #{get-hsl(map.get($base-colors, foreground-primary))};
-  --iui-color-foreground-positive-hsl: #{get-hsl(map.get($base-colors, foreground-positive))};
-  --iui-color-foreground-warning-hsl: #{get-hsl(map.get($base-colors, foreground-warning))};
-  --iui-color-foreground-negative-hsl: #{get-hsl(map.get($base-colors, foreground-negative))};
+  // #region Background variables
+  --iui-color-background: #{hex($bg-7)};
+  --iui-color-background-zebra: #{hex($bg-8)};
+  --iui-color-background-hover: #{hex($bg-9)};
+  --iui-color-background-backdrop: #{hex($bg-5)};
+  --iui-color-background-backdrop-hover: #{hex($bg-6)};
+  --iui-color-background-disabled: #{hex($bg-4)};
+  --iui-color-background-primary: #{hex($bg-primary-1)};
+  --iui-color-background-primary-hover: #{hex($bg-primary-2)};
+  --iui-color-background-primary-muted: #{hex($bg-primary-0)};
+  --iui-color-background-positive: #{hex($bg-positive-1)};
+  --iui-color-background-positive-hover: #{hex($bg-positive-2)};
+  --iui-color-background-positive-muted: #{hex($bg-positive-0)};
+  --iui-color-background-warning: #{hex($bg-warning-1)};
+  --iui-color-background-warning-hover: #{hex($bg-warning-2)};
+  --iui-color-background-warning-muted: #{hex($bg-warning-0)};
+  --iui-color-background-negative: #{hex($bg-negative-1)};
+  --iui-color-background-negative-hover: #{hex($bg-negative-2)};
+  --iui-color-background-negative-muted: #{hex($bg-negative-0)};
+  --iui-color-background-transparent-hover: #{hsl(0 0% 100% / 0.065)};
+  // #endregion
+  // #region Border variables
+  --iui-color-border-subtle: #{hex($bg-9)};
+  --iui-color-border: #{hex($bg-10)};
+  --iui-color-border-foreground: #{hex($fg-8)};
+  --iui-color-border-foreground-hover: #{hex($fg-6)};
+  --iui-color-border-disabled: #{hex($bg-9)};
+  --iui-color-border-primary: #{hex($fg-primary-1)};
+  --iui-color-border-positive: #{hex($fg-positive-1)};
+  --iui-color-border-warning: #{hex($fg-warning-1)};
+  --iui-color-border-negative: #{hex($fg-negative-1)};
+  // #endregion
+  // #region Icon variables
+  --iui-color-icon: #{hex($fg-3)};
+  --iui-color-icon-hover: #{hex($fg-2)};
+  --iui-color-icon-muted: #{hex($fg-7)};
+  --iui-color-icon-muted-hover: #{hex($fg-6)};
+  --iui-color-icon-disabled: #{hex($fg-8)};
+  --iui-color-icon-primary: #{hex($fg-primary-2)};
+  --iui-color-icon-positive: #{hex($fg-positive-2)};
+  --iui-color-icon-warning: #{hex($fg-warning-2)};
+  --iui-color-icon-negative: #{hex($fg-negative-2)};
+  // #endregion
+  // #region Text variables
+  --iui-color-text: #{hex($fg-2)};
+  --iui-color-text-hover: #{hex($fg-1)};
+  --iui-color-text-muted: #{hex($fg-5)};
+  --iui-color-text-disabled: #{hex($fg-7)};
+  --iui-color-text-primary: #{hex($fg-primary-1)};
```

Comment:
I'm wondering if we should introduce a new set of aliases for `accent`, to separate it from status colors. Or we could rename `primary` to `accent`. I think it would be nice for customizability.